clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1533,11 +1552,16 @@ private ClusterResourceListeners configureClusterResourceListeners(Deserializer<
         return clusterResourceListeners;
     }
 
-    private void close(boolean swallowException) {
+    private void close(long timeoutMs, boolean swallowException) {
         log.trace("Closing the Kafka consumer.");
         AtomicReference<Throwable> firstException = new AtomicReference<>();
         this.closed = true;
-        ClientUtils.closeQuietly(coordinator, "coordinator", firstException);
+        try {
+            coordinator.close(timeoutMs, requestTimeoutMs);
+        } catch (Throwable t) {
+            firstException.compareAndSet(null, t);
+            log.error("Failed to close coordinator", t);
```

Comment:
`ClientUtils.closeQuietly` which was used earlier logs `error`. Since that is used for the other entities in this method, it makes sense to log at the same level. Exceptions do indicate a catastrophic failure (broker unavailable etc. close without exceptions).